sql/core/src/test/scala/org/apache/spark/sql/DataFrameSuite.scala

```diff
@@ -641,6 +641,10 @@ class DataFrameSuite extends QueryTest with SharedSQLContext {
         Row(key, value, key + 1)
       }.toSeq)
     assert(df.schema.map(_.name) === Seq("key", "valueRenamed", "newCol"))
```

Comment:
@jaceklaskowski: I agree with you that the code can be made shorter and more elegant in the ways that you pointed out. Nonetheless, the improvements: (1) removing () in toDF() and (2) replacing the schema map with fieldNames, are actually applicable to other places in the file "DataFrameSuite.scala" as well. In other words, it is not directly related to this PR. I think the proper way to handle this is rather in another JIRA issue. Feel free to fire it.